v-next/hardhat-keystore/src/internal/utils/validate-key.ts

```diff
@@ -0,0 +1,9 @@
+export async function validateKey(key: string): Promise<boolean> {
+  const KEY_REGEX = /^[a-zA-Z_]+[a-zA-Z0-9_]*$/;
```

Comment:
```suggestion
  TODO: Revist this once encryption is implemented.
  const KEY_REGEX = /^[a-zA-Z_]+[a-zA-Z0-9_]*$/;
```

Reply:
I'm not even sure if we should validate that. Special chars are good for passwords.

Reply:
This regex is for the KEY validation (same naming convention as ENV variables), not the password. The password regex is [this one](https://github.com/NomicFoundation/hardhat/pull/5654/files#diff-5d0d1b07b3cc14c33477602b9a9261a21abf6ee2ef0a2774c98a64b5fb7f87be)

Reply:
ohhh, got it!